src/WebJobs.Script/Config/HostConfigurationProfile.cs

```diff
@@ -30,6 +30,8 @@ public sealed class HostConfigurationProfile
                     ConfigurationSectionNames.CustomHandler, "enableHttpProxyingRequest"), "true"),
                 KeyValuePair.Create(ConfigurationPath.Combine(
                     ConfigurationSectionNames.Http, "routePrefix"), string.Empty),
+                KeyValuePair.Create(ConfigurationPath.Combine(
+                    ConfigurationSectionNames.CustomHandler, "http", "Routes", "0", "route"), "{*route}"),
```

Comment:
Casing doesn't really matter, but I think lowercase would be nice for consistency.

```suggestion
                    ConfigurationSectionNames.CustomHandler, "http", "routes", "0", "route"), "{*route}"),
```